crates/payload/builder/src/payload.rs

```diff
@@ -129,6 +129,8 @@ pub struct PayloadBuilderAttributes {
     pub prev_randao: H256,
     /// Withdrawals for the generated payload
     pub withdrawals: Vec<Withdrawal>,
+    /// Root of the parent beacon block
+    pub parent_beacon_block_root: H256,
```

Comment:
This needs to be an `Option<H256>` so header conversions can preserve `None` pre-cancun